blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for the blog app."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
```

Comment:
@Martolivna I want you to write tests for this model.

Create a `tests/test_models.py` file, import this model there, follow https://docs.djangoproject.com/en/4.2/topics/testing/overview/ to declare a test case class and make dedicated methods for every separate thing/aspect you're testing.
Generally, those tests would make an instance of this class and then, call its methods + check their return values or side effects.

Make sure to call every method you declared in this class.

Bonus: you can also integrate coverage reporting like so https://adamj.eu/tech/2019/04/30/getting-a-django-application-to-100-percent-coverage/.